### PR TITLE
Add Prism Launcher (Minecraft) blogpost

### PR DESCRIPTION
**Pull request by TheEgghead27**

### CODE REVIEW

**Comment by TheEgghead27** [comment]
WIP w/ @thegu5

**Comment by TheEgghead27** [comment]
*known bug: current rendering does not give id to headers

File: _posts/minecraft-prismlauncher.md [hunk 1]
```diff
@@ -0,0 +1,159 @@
...
+### Logging into Minecraft
+PrismLauncher isn't a piracy tool! You still need to own Minecraft to use it!
+
+Click the accounts button in the top-right corner and head to "Manage Accounts":
+
+![Menu that appears when "Accounts" - denoted with a gray Steve face icon, is clicked. There is No Default Account, and an option to Manage Accounts.](/img/posts/minecraft-prismlauncher/acc.png)
+
+![The Account Settings menu. It reads "Welcome! If you're new here, you can click the "Add" button to add your Mojang or Minecraft account.](/img/posts/minecraft-prismlauncher/account-settings.png)
+
+Use the sign in options on the right-hand side of the menu to sign in to your Minecraft account (Microsoft or Mojang).
```
Comment (thegu5) [inline _posts/minecraft-prismlauncher.md]:
```suggestion
Use the sign in options on the right-hand side of the menu to sign in with your Minecraft (Microsoft) account. Although an option for Mojang accounts is provided, they have been phased out - so if you still have one, follow [this guide from Microsoft](https://help.minecraft.net/hc/en-us/articles/4403181904525).
```

File: _posts/minecraft-prismlauncher.md [hunk 2]
```diff
@@ -0,0 +1,159 @@
...
+
+![Add Instance - Vanilla](/img/posts/minecraft-prismlauncher/vanilla.png)
+
+Scroll down to pick the version you want, and hit OK.
+You do not need to select a mod loader just yet, but if you know what you're doing, you can enable one here.
+
+You can select it and hit the play button, and then it will download the Minecraft date from Mojang, and launch it!
+
+### Installing Performance Mods
+#### OptiFine
```
Comment (thegu5) [inline _posts/minecraft-prismlauncher.md:102]: This optifine jar can also be used as a forge mod, which is simpler (?).

File: _posts/minecraft-prismlauncher.md [hunk 3]
```diff
@@ -0,0 +1,159 @@
...
+
+![The Version tab. "Add to Minecraft.jar" is highlighted.](/img/posts/minecraft-prismlauncher/add.png)
+
+You are done, and can now launch with OptiFine.
+
+#### Chem Lab
+> *honestly at this point fabric jar with lithium and phosphor*
+\- pcao#2918
+
+A modern way to optimize Minecraft is to use the Fabric mod launcher with tools like Sodium, Lithium, and Phosphor, along with Iris Shaders to provide compatibility with OptiFine shaders.
```
Comment (thegu5) [inline _posts/minecraft-prismlauncher.md]:
```suggestion
A modern way to optimize Minecraft is to use the Fabric mod loader with tools like Sodium, Lithium, and Phosphor, along with Iris Shaders to provide compatibility with OptiFine shaders.
```

File: _posts/minecraft-prismlauncher.md [hunk 4]
```diff
@@ -0,0 +1,159 @@
...
+In fact, these are some of the top mods on [modrinth](https://modrinth.com/mods).
+
+Prism Launcher makes our lives easy by giving us an easy way to install Fabric and mods off of modrinth.
+
+First, right-click and edit the instance settings.
```
Comment (thegu5) [inline _posts/minecraft-prismlauncher.md:139]: Could also make this a new instance (show off how easy it is to have multiple instances side by side?)

**Review by thegu5 — CHANGES_REQUESTED** [review]

File: _posts/minecraft-prismlauncher.md (hunk 2 above)
Comment (TheEgghead27) [reply _posts/minecraft-prismlauncher.md:102]: True but it isn't on CurseForge so it'd be just mediocre

File: _posts/minecraft-prismlauncher.md (hunk 4 above)
Comment (TheEgghead27) [reply _posts/minecraft-prismlauncher.md:139]: Added

File: _posts/minecraft-prismlauncher.md [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+Prism Launcher will then download the mods, and handle installing them for you!
+
+You can now launch the game, and enjoy your boosted performance!
+
+Have fun!
+
+### Date Published: 2023_04_16 (YYYY_MM_DD)
+
+### Written by: David Chen [@TheEgghead27](https://github.com/TheEgghead27)
+### Reviewed by: {gus} [@thegu5](https://github.com/thegu5)
```
Comment (TheEgghead27) [inline _posts/minecraft-prismlauncher.md]: Oh @thegu5 lmk how you want your name to look down here

Comment (thegu5) [inline _posts/minecraft-prismlauncher.md]:
```suggestion
### Reviewed by: Gus [@thegu5](https://github.com/thegu5)
```

**Review by thegu5 — APPROVED** [review]

**Comment by TheEgghead27** [comment]
@leomet07 have at it! lemme know when it's up so i can announce on the proper channels

**Review by leomet07 — APPROVED** [review]
LGTM! nice work !

**Pull request merged**